src/components/filters/useLastSeenFilter.js

```diff
@@ -92,41 +92,41 @@ export const useLastSeenFilter = (
   };
 
   //This date comes from patternfly component. This manages the 1st date picker
-  const onFromChange = (date) => {
+  const onFromChange = (_event, str, date) => {
```

Comment:
The renaming it to `str` isn't really needed, or?

Reply:
I was mostly just following the convention in the doc, I could change it